src/options.ts

```diff
@@ -191,6 +191,22 @@ function applyHttpOptionsDefaults(options: N9NodeRouting.Options): void {
 	}
 }
 
+function applyHttpClientOptionsDefaults(options: N9NodeRouting.Options): void {
+	// Defaults options for http client
+	const defaulSensitiveHeadersOptions: N9NodeRouting.HttpClientSensitiveHeadersOptions = {
```

Comment:
The check of env is missing to manage NODE_ENV=development

Check how it is done elsewhere.

Reply:
@BenjD90 done!